### PR TITLE
Update sparkbox

### PR DESCRIPTION
**Pull request by scribblemaniac**
Switch to versioned url

### CODE REVIEW

File: Casks/sparkbox.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
 cask :v1 => 'sparkbox' do
-  version :latest
-  sha256 :no_check
+  version '1.2.4'
+  sha256 '1b5eb4fb54de0e3151405ab3f8d643b13188a68ea4a1a2a2a2f552cf7f138b30'
 
-  url 'http://t.icyblaze.com/sblatest'
-  appcast 'http://matrix.icyblaze.com/index.php/checkupdate/p/8'
+  url "http://s3.amazonaws.com/IcyBlaze-iDocument2/Download/sparkbox_#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/sparkbox.rb]: Missing the comment.

Comment (scribblemaniac) [inline Casks/sparkbox.rb]: Oops

**Comment by scribblemaniac** [comment]
Fixed

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 2068499062232fcd3477d0c1a12c8501a838cbe4 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**